packages/cli/bin/download-connector-list.js

```diff
@@ -0,0 +1,56 @@
+const https = require('https');
+const fs = require('fs');
+const path = require('path');
+const util = require('util');
+
+const readFileAsync = util.promisify(fs.readFile);
+const writeFileAsync = util.promisify(fs.writeFile);
+
+const dest = path.resolve('generators/datasource/connectors.json');
+const URL =
+  'https://raw.githubusercontent.com/strongloop/loopback-workspace/master/available-connectors.json';
+
+/**
+ * Function to dowload the list of available connectors from loopback-workspace
+ * so the list only has to be maintained in one place.
+ */
+async function download() {
+  var file = fs.createWriteStream(dest);
+  var request = https
+    .get(URL, function(response) {
+      response.pipe(file);
+      file.on('finish', function() {
+        file.close();
+        transformConnectorJSON();
+      });
+    })
+    .on('error', function(err) {
+      fs.unlink(dest);
+      return err;
+    });
+}
+
+async function transformConnectorJSON() {
```

Comment:
Can we come up a json format to have more metadata about the prompts?

Reply:
Not sure what you mean -- using the same format as `loopback-workspace` but transformed to be simpler for consumption by CLI prompts -- in LB3 the CLI does the transformation when the command is called, I am preferring to do it whenever we get the file.